runtime/src/lib.rs

```diff
@@ -142,7 +146,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     //   `spec_version`, and `authoring_version` are the same between Wasm and native.
     // This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
     //   the compatible custom types.
-    spec_version: 196,
+    spec_version: 202,
```

Comment:
Intentional?

Reply:
I'd remove the change. This was +1 to Finney version, but we will be updating it to the correct one anyway later.